### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by HenryNovoa**

### CODE REVIEW

File: staff/henry-novoa/cv-styled/index.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+</head>
+<body>
+    <section class="box info">
+      
+      <h2 class="personalHeader">Personal Data</h2>
+      <p class="name"> Name: Henry Novoa</p>
+      <address>
+          Date of birth: 8th of June, 1990 <br> 
+          Place of birth: New York, USA    <br>
+          Current location: Barcelona      <br>
```
Comment (aleixcam) [inline staff/henry-novoa/cv-styled/index.html]: use p tags for newlines instead of br

File: staff/henry-novoa/cv-styled/index.html [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Curriculum vitae</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="box info">
+      
+      <h2 class="personalHeader">Personal Data</h2>
```
Comment (Arcoders) [inline staff/henry-novoa/cv-styled/index.html]: as a good practice the classes in css are named separated by (-) like : personal-header

File: staff/henry-novoa/cv-styled/index.html [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+      <p class="name"> Name: Henry Novoa</p>
```
Comment (Pacoscript) [inline staff/henry-novoa/cv-styled/index.html:15]: I would put the h2 "personalHeader" inside the tag address. And each element inside "address" tag, inside a "p" tag for example.

File: staff/henry-novoa/cv-styled/index.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Curriculum vitae</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="box info">
```
Comment (Pacoscript) [inline staff/henry-novoa/cv-styled/index.html:12]: You could try to add a photograph to the right of the "box info"

File: staff/henry-novoa/cv-styled/index.html (hunk 1 above)
Comment (HenryNovoa) [reply staff/henry-novoa/cv-styled/index.html]: I'm on it.

File: staff/henry-novoa/cv-styled/index.html (hunk 2 above)
Comment (HenryNovoa) [reply staff/henry-novoa/cv-styled/index.html]: I'm on it

File: staff/henry-novoa/cv-styled/index.html (hunk 3 above)
Comment (HenryNovoa) [reply staff/henry-novoa/cv-styled/index.html:15]: For how it is structured, I want the h2 outside of address. I will put each element inside address with a p

File: staff/henry-novoa/cv-styled/index.html (hunk 4 above)
Comment (HenryNovoa) [reply staff/henry-novoa/cv-styled/index.html:12]: Will add if I have time, adding an image requires also  reconfiguring the structure

**Pull request merged**